### ISSUE TITLE
Is it possible to import a CA certificate (pem file) into min browser?  In order to use min behind the proxy,  I need this functionality.. 

### ISSUE THREAD

**Issue by arafune** [minbrowser/min#2070]

**Comment by PalmerAL** [comment]
No, it's not, sorry. You can launch Min temporarily with `--ignore-certificate-errors`, but that will disable certificate validation completely, so you shouldn't use that for normal browsing.

**Comment by arafune** [comment]
Ah, that's a shame.  Do you have any plans to implement it? Or is it really difficult task in the current situation?
